tools/devops/automation/build-pipeline.yml

```diff
@@ -27,10 +211,60 @@ trigger:
       - msbuild/Xamarin.Localization.MSBuild/README.md
 
 extends:
-  template: templates/pipelines/build-pipeline.yml
+  template: azure-pipelines/MicroBuild.1ES.Official.yml@CustomPipelineTemplates
   parameters:
-    isPR: false
-    signingSetupSteps: 
-      - template: ./templates/sign-and-notarized/setup.yml
-        parameters:
-          isPR: false
+    pool:  # default pool to be used forvalidation jobs
+      name: AzurePipelines-EO
+      image: 1ESPT-Windows2022
+      os: windows
+    sdl:
+      baseline:
+        baselineFile: '$(System.DefaultWorkingDirectory)\\maccore\\tools\\devops\\baselines.gdnbaselines'
+      suppression:
+        suppressionFile: '$(System.DefaultWorkingDirectory)\\maccore\\tools\\devops\\suppress.gdnsuppress'
```

Comment:
Can we put these files in xamarin-macios instead? To reduce the dependency on maccore...

Reply:
Will do that in a following PR.